port_ocean/cli/cookiecutter/hooks/post_gen_project.py

```diff
@@ -2,10 +2,14 @@
 
 
 def delete_docker_files():
-    if "{{ cookiecutter.remove_docker_files }}" == "True":
+    if "{{ cookiecutter.remove_docker_files }}" == "true":
```

Comment:
Please figure out what is the correct way to reference the value of these booleans from cookiecutter